.github/workflows/pr-checks.yml

```diff
@@ -0,0 +1,24 @@
+name: tests
+
+on:
+  pull_request:
+    branches: [master]
```

Comment:
```suggestion
```

We should run it on all PRs

Reply:
https://github.com/AirHelp/treasury/pull/180/commits/37eaca988e97812a57026acc26b5f3fce6bf81d6 addressed